From f149693ceeed469658e32b3ac5a9e5357b1164d3 Mon Sep 17 00:00:00 2001
From: Wu Liangqing <wlq@rock-chips.com>
Date: Wed, 17 Jun 2020 17:28:41 +0800
Subject: [PATCH] arm64: rockchip_defconfig: add sensors for rk3368a tablet

+CONFIG_TOUCHSCREEN_GSL3676=y
+CONFIG_GS_SC7660=y
+CONFIG_GS_SC7A20=y
+CONFIG_GS_DA223=y
+CONFIG_VIDEO_OV2680=y
+CONFIG_VIDEO_GC032A=y

Change-Id: Ifb6eac8e1f36fad84a911efb09e23b8ec9bd5c57
Signed-off-by: Wu Liangqing <wlq@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index 6774fd9da073..21ac78ca2551 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -431,6 +431,7 @@ CONFIG_TOUCHSCREEN_GSLX680_PAD=y
 CONFIG_TOUCHSCREEN_GSLX680_VR=y
 CONFIG_TOUCHSCREEN_GSL3673=y
 CONFIG_TOUCHSCREEN_GSL3673_800X1280=y
+CONFIG_TOUCHSCREEN_GSL3676=y
 CONFIG_TOUCHSCREEN_GT9XX=y
 CONFIG_TOUCHSCREEN_HYN_CST2XX=y
 CONFIG_TOUCHSCREEN_GT1X=y
@@ -445,11 +446,14 @@ CONFIG_GS_KXTJ9=y
 CONFIG_GS_LIS3DH=y
 CONFIG_GS_MMA7660=y
 CONFIG_GS_MC3230=y
+CONFIG_GS_SC7660=y
+CONFIG_GS_SC7A20=y
 CONFIG_GS_SC7A30=y
 CONFIG_GS_MXC6655XA=y
 CONFIG_GS_LSM303D=y
 CONFIG_LSM330_ACC=y
 CONFIG_BMA2XX_ACC=y
+CONFIG_GS_DA223=y
 CONFIG_COMPASS_DEVICE=y
 CONFIG_COMPASS_AK8975=y
 CONFIG_COMPASS_AK8963=y
@@ -538,10 +542,12 @@ CONFIG_V4L_PLATFORM_DRIVERS=y
 CONFIG_VIDEO_ROCKCHIP_ISP1=y
 # CONFIG_MEDIA_SUBDRV_AUTOSELECT is not set
 CONFIG_VIDEO_VM149C=y
+CONFIG_VIDEO_OV2680=y
 CONFIG_VIDEO_OV5695=y
 CONFIG_VIDEO_OV8858=y
 CONFIG_VIDEO_OV13850=y
 CONFIG_VIDEO_GC0312=y
+CONFIG_VIDEO_GC032A=y
 CONFIG_VIDEO_GC2145=y
 CONFIG_VIDEO_GC2355=y
 CONFIG_VIDEO_SGM3784=y
-- 
2.35.3

